mne/channels/montage.py

```diff
@@ -206,6 +206,16 @@ def read_montage(kind, ch_names=None, path=None, unit='m', transform=False):
         ch_names_ = []
         pos = []
         with open(fname) as fid:
+            # Default units are meters
+            scale_factor = 1.
+            for line in fid:
+                if 'UnitPosition' in line:
+                    units = line.split()[1]
+                    if units == 'm':
+                        break
+                    elif units == 'mm':
+                        scale_factor = 1. / 1000.
+                        break
```

Comment:
Excellent, I knew there was a better way to do that.

Unfortunately that will error out if the first line doesn't have the units in it -- so [this line](https://github.com/mne-tools/mne-python/blob/master/mne/channels/data/montages/standard_1005.elc#L1-L2) will hit the runtime error.  Instead in this last commit I just said "if you've hit `Positions` you've gone too far."  What do you think, is there a better way?